components/org.wso2.carbon.identity.application.authenticator.google/src/main/java/org/wso2/carbon/identity/application/authenticator/google/GoogleErrorConstants.java

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright (c) 2019, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
Update the license header

Reply:
Fixed f827401b9ec0ac217c512006559e574c77e1a998